README.md

```diff
@@ -0,0 +1,49 @@
+OSIC Reference implementation
+=============================
+
+# What is it?
+
+Reference implementation is opinionated way to deploy OpenStack. One of big
```

Comment:
Forcing our opinions, take it or leave it approach with caveat of howtos about overriding our decisions, but with "you are on your own" clause